### PR TITLE
42

### PR DESCRIPTION
**Pull request by Mipronimo**
@plettich please do not approve ;)

### CODE REVIEW

**Comment by plettich** [comment]
should i add a comment only or do nothing at all?

File: hello.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 print("Hello world! bye")
-print("The result is", 11)
\ No newline at end of file
+print("The result is", 42)
+print("whatever")
```
Comment (fredreichbier) [inline hello.py]: Please capitalize.

**Comment by Mipronimo** [comment]
> should i add a comment only or do nothing at all?

@plettich you can just ignore it and put your emails to your spam directory :D

File: hello.py (hunk 1 above)
Comment (Mipronimo) [reply hello.py]: fix in 345b23c

**Pull request merged**